test/common/common/log_macros_test.cc

```diff
@@ -101,7 +101,7 @@ TEST(Logger, checkLoggerLevel) {
     }
   };
 
-  logTestClass testObj;
+  LogTestClass testObj;
```

Comment:
s/testObj/test_obj/ I think you'll catch this in next identifier-naming PR but feel free to fix here too.